### ISSUE TITLE
Implement KB Landing Page Redesign 

### ISSUE THREAD

**Issue by gracekretschmer-metrostar** [department-of-veterans-affairs/va.gov-cms#17940]
## User Story or Problem Statement
The knowledge base landing page prototype needs to be implemented into production.

As an editor, I want to navigate to the Knowledge Base landing page and feel empowered to find any/all of the information I require to continue to complete my editor-based goals.

### The Editor-centered Design Principles serve as our team's NorthStar:
**Purpose-driven:** Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
**Efficient:** Remove distractions and create clear, straightforward paths to get the job done.
**Approachable:** Offer friendly guidance over authoritative instruction.
**Consistent:** Reduce users’ mental load by allowing them to fall back on pattern recognition to complete tasks.
**Empowering:** Provide clear information to help editors make decisions about their work.

## Description or Additional Context 
The Knowledge Base is undergoing a redesign, starting first with the landing page. 
- The landing page is the first page where the editor lands when seeking 'editor support'.
- The current design pushes pertinent links below the fold, often hiding the resources required.
- The current search functionality is lacking in terms of effectiveness; we are tackling this portion in a separate ticket.
- The KB taxonomy is not inclusive and therefore will require additional work to organize KB content into logical groupings

We have redesigned the landing page to better support the editor by:
- Taking inspiration from the [veteran-facing support experience](https://www.va.gov/resources/)
- We are adding sidebar columns that create a visual path from top to bottom, allowing the eye an opportunity to rest within the given white space. The sidebar layout will be carried into the internal templates creating consistency.
- We will utilize Help Desk support data to better understand the links that are most supportive to the larger editor group. 

## Reference Links
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17383

## Steps for Implementation
- [ ] Design Proto-type has been approved by PO, PM, and Lead Engineer 
- [ ] Developer Review of the KB landing page prototype to determine technical path forward for implementation.
- [ ] Develop and QA Test for UI/UX expectation - identifying any/all Accessibility concerns

## Acceptance Criteria
- [ ] The KB landing page is implemented that is a reflection of the KB prototype.

**Comment by gracekretschmer-metrostar** [comment]
Next steps: this will be Jake's priority this week. Will need support in understanding how to navigate committing code and workflow within Github from @edmund-dunn.

**Comment by EWashb** [comment]
Is this blocked moving forward on the dev pieces @gracekretschmer-metrostar ?

**Comment by gracekretschmer-metrostar** [comment]
We are blocked on dev capacity. Jake's priority this sprint is supporting some drupal work for Next Build (https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14052), so I told him to pick up this knowledge base work once he is done with the Next Build work. Jake is scrubbing in on this work because Edmund is out.

**Comment by gracekretschmer-metrostar** [comment]
Jake: could use help with getting unblocked on cypress tests failing.

**Comment by gracekretschmer-metrostar** [comment]
Jake: need @timcosgrove's support on the URL pathways.
